api_tests/Test.cs

```diff
@@ -175,6 +175,57 @@ public void TestDeviceCapabilityQueries()
             Assert.AreEqual(set.Device as IReadable<Light> != null, set.Result);
         }
     }
+
+	[Test()]
+	public void TestDeviceVars()
+	{
+		List<Device> devices = new List<Device>()
+		{
+			new AlarmSystem(null, null, null),
+			new CeilingFan(null, null, null),
+			new GarageDoor(null, null, null),
+			new LightSwitch(null, null, null),
+			new Thermostat(null, null, null)
+		};
+
+		foreach (var device in devices)
+		{
+			Assert.IsTrue(device.ID.HouseID == 0 &&
+						  device.ID.RoomID == 0 &&
+						  device.ID.DeviceID == 0);
+		}
+
+		foreach (var device in devices)
+		{
+			device.ID.HouseID = 1;
+			device.ID.RoomID = 2;
+			device.ID.DeviceID = 3;
+		}
+
+		foreach (var device in devices)
+		{
+			Assert.IsTrue(device.ID.HouseID &&
```

Comment:
Xamarin seems terrible with handling NuGet packages. If you look in repositories, usually there will be local mods showing how it broke the packages, and you are able to help remove those issues.
> On Apr 20, 2015, at 4:59 PM, willrd9 <notifications@github.com> wrote:
> 
> In api_tests/Test.cs <https://github.com/ECE4574-5574/Devices/pull/43#discussion_r28728662>:
> 
> > +		{
> > +			Assert.IsTrue(device.ID.HouseID == 0 &&
> > +						  device.ID.RoomID == 0 &&
> > +						  device.ID.DeviceID == 0);
> > +		}
> > +
> > +		foreach (var device in devices)
> > +		{
> > +			device.ID.HouseID = 1;
> > +			device.ID.RoomID = 2;
> > +			device.ID.DeviceID = 3;
> > +		}
> > +
> > +		foreach (var device in devices)
> > +		{
> > +			Assert.IsTrue(device.ID.HouseID &&
> I can change it to be more explicit but I'm sorry for the obvious error. Ever since I synced I can't complie to actually see if anything works. After I clone a brand new copy of automation_system and follow your testing instructions on the wiki, I'm still having about 90 errors attributed to references. Every nuget package is installed properly.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/ECE4574-5574/Devices/pull/43/files#r28728662>.
>